css/.scss-lint.yml

```diff
@@ -12,6 +12,11 @@ linters:
         space_before_bang: true
         space_after_bang: false
 
+    # Checks for BEM selectors with more elements than a specified maximum number.
+    BemDepth:
+        enabled: false
```

Comment:
Can we turn this on? Would be a helpful check for folks just just starting with BEM.